App.tsx

```diff
@@ -42,12 +43,13 @@ const App: React.FC = () => {
   const [contacts, setContacts] = React.useState<ContactInterface[]>([]);
   const [notes, setNotes] = React.useState<NoteInterface[]>([]);
   const [events, setEvents] = React.useState<EventInterface[]>([]);
-  const [user, setUser] = React.useState<UserInterface>(null);
+  const [user, setUser] = React.useState<UserInterface | null>(null);
   const [beneficiaries, setBeneficiaries] = React.useState<BeneficiaryInterface[]>([]);
-  const [beneficiary, setBeneficiary] = React.useState<UserInterface>(null);
-  const [lastUsername, setLastUsername] = React.useState<string>(null);
+  const [beneficiary, setBeneficiary] = React.useState<UserInterface | null>(null);
+  const [lastUsername, setLastUsername] = React.useState<string | null>(null);
   const [theme, themeActions] = useBoolean(false);
-
+  const [attempts, setAttempts] = React.useState<number>(0);
+  const [isTemporarlyBlocked, setIsTemporarlyBlocked] = React.useState<boolean>(false);
```

Comment:
It may be a bad idea, but I wonder if we could implement our own functions here to have all our blocking logic here, something as
```jsx
const isTemporarlyBlocked = () => attempts === LoginTemporisationContext.MAX_LOGIN_ATTEMPTS;
const setIsTemporarlyBlocked = () => setAttempts(10);
```